plugins/outputs/sql/sql.go

```diff
@@ -48,21 +48,23 @@ type ConvertStruct struct {
 }
 
 type SQL struct {
-	Driver                string          `toml:"driver"`
-	DataSourceName        string          `toml:"data_source_name"`
-	TimestampColumn       string          `toml:"timestamp_column"`
-	TableTemplate         string          `toml:"table_template"`
-	TableExistsTemplate   string          `toml:"table_exists_template"`
-	InitSQL               string          `toml:"init_sql"`
-	Convert               ConvertStruct   `toml:"convert"`
-	ConnectionMaxIdleTime config.Duration `toml:"connection_max_idle_time"`
-	ConnectionMaxLifetime config.Duration `toml:"connection_max_lifetime"`
-	ConnectionMaxIdle     int             `toml:"connection_max_idle"`
-	ConnectionMaxOpen     int             `toml:"connection_max_open"`
-	Log                   telegraf.Logger `toml:"-"`
+	Driver                   string          `toml:"driver"`
+	DataSourceName           string          `toml:"data_source_name"`
+	TimestampColumn          string          `toml:"timestamp_column"`
+	TableTemplate            string          `toml:"table_template"`
+	TableExistsTemplate      string          `toml:"table_exists_template"`
+	TableUpdateTemplate      string          `toml:"table_update_template"`
+	TableListColumnsTemplate string          `toml:"-"`
```

Comment:
Do not export this (i.e. start with lowercase letter) and move it down to the internal variables.